core/src/main/java/greencity/dto/tipsandtricks/TipsAndTricksDto.java

```diff
@@ -0,0 +1,45 @@
+package greencity.dto.tipsandtricks;
+
+import greencity.dto.user.TipsAndTricksAuthorDto;
+import java.time.ZonedDateTime;
+import java.util.List;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@ToString(exclude = "author")
+@EqualsAndHashCode
+public class TipsAndTricksDto {
```

Comment:
AddTipsAndTricksDtoRequest , AddTipsAndTricksDtoResponse  and TipsAndTricksDto  - looks really similar to me. I think you can leave only TipsAndTricksDto  class not to duplicate all variables.

Reply:
This will also decreases the number of converters

Reply:
If you see that you really need more than one class - think about some general Abstract class with 2 concrete child classes to avoid duplication

Reply:
Replaced TipsAndTricksAuthorDto with AuthorDto to be further used with EcoNews too.
Removed TipsAndTricksDto, left 2 DTOs: TipsAndTricksDtoRequest and TipsAndTricksDtoResponse